terraform/README.md

```diff
@@ -0,0 +1,41 @@
+# Airbyte Terraform
+
+## Connector Development Infrastructure
+We use Terraform to manage any persistent infrastructure used for developing or testing connectors.
+
+### Workflow
+
+#### Setup Credentials
+**GCP**
+
+Copy the contents of the Lastpass credentials `Connector GCP Terraform Key` into `gcp/connectors/secrets/svc_account_creds.json`. 
+
+Any `secrets` directory in the entire repo is gitignored by default so there is no danger of checking credentials into git.  
+
+**AWS**
+
+Coming soon. 
+
+**Azure**
+
+Coming soon. 
+
+#### Iteration Cycle
+To run terraform commands use the TF wrapper: 
+
+```
+# From the airbyte repo root
+./tools/terraform/terraform.sh ./terraform/<cloud-provider>/connectors/ <TF command>
+```
+
+_hint_: alias `atf` (short for `airbyte-terraform`) to `./tools/terraform/terraform.sh` to 10x your iteration speed.    
+
+**If this is your first time running Terraform** run the `init` command 
```

Comment:
For all of these commands please give the full command so there's no ambiguity `./tools/terraform/terraform.sh init`, `./tools/terraform/terraform.sh apply`, etc. It also makes it copy-pastable.